src/main/java/seedu/address/logic/parser/EditCommandParser.java

```diff
@@ -52,6 +54,11 @@ public EditCommand parse(String args) throws ParseException {
             editPersonDescriptor.setEmail(ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get()));
         }
 
+        if (argMultimap.getValue(PREFIX_GIT_USERNAME).isPresent()) {
+            editPersonDescriptor.setUsername(ParserUtil
```

Comment:
Should it be updated to this instead for consistency?
```suggestion
            editPersonDescriptor.setGitUsername(ParserUtil
```

Reply:
Noted with thanks :). I will update it in the next commit as it affect a few classes